assets/app/scripts/directives/metrics.js

```diff
@@ -169,83 +205,104 @@ angular.module('openshiftConsole')
           return null;
         }
 
-        function updateChart(data, metric) {
-          var dates = ['dates'], values = [metric.units];
-          var usage = scope.usageByMetric[metric.id] = {
-            total: getLimit(metric)
-          };
+        function updateChart(metric) {
+          var dates, values = {};
 
-          var mostRecentValue = data[data.length - 1].value;
-          if (isNaN(mostRecentValue)) {
-            mostRecentValue = 0;
-          }
-          if (metric.id === 'memory/usage') {
-            mostRecentValue = bytesToMiB(mostRecentValue);
-          }
+          angular.forEach(metric.datasets, function(dataset) {
+            var metricID = dataset.id, metricData = dataset.data;
 
-          // Round to the closest whole number for the utilization chart.
-          usage.used = d3.round(mostRecentValue);
-          if (usage.total) {
-            usage.available = Math.max(usage.total - usage.used, 0);
-          }
+            dates = ['dates'], values[metricID] = [dataset.label || metricID];
+
+            var usage = scope.usageByMetric[metricID] = {
+              total: getLimit(metricID)
+            };
+
+            var mostRecentValue = metricData[metricData.length - 1].value;
+            if (isNaN(mostRecentValue)) {
+              mostRecentValue = 0;
+            }
+            if (/network\/rx|tx/.test(metricID) || metricID === 'memory/usage') {
+              mostRecentValue = bytesToMiB(mostRecentValue);
+            }
```

Comment:
Maybe we should add a property to `$scope.metrics` that says if it's a memory metric, then use that to decide whether to use `bytesToMiB` (here and below).